### PR TITLE
Wait for REST server start in start_server_single_model_onnx fixture 

### PR DESCRIPTION
**Pull request by mzegla**

### CODE REVIEW

**Review by mkuczyns11 — APPROVED** [review]

**Review by atobiszei — COMMENTED** [review]
is it the only test that uses REST API?

**Comment by atobiszei** [comment]
Seems to me that we should make this change for all REST related tests

File: tests/functional/object_model/docker.py [hunk 1]
```diff
@@ -106,8 +106,13 @@ def get_logs(self):
 
     def ensure_logs(self):
         logs = self.get_logs()
-        if self.container_log_line not in logs:
-            assert False, f"Not found required phrase {self.container_log_line}"
+        if type(self.container_log_line) == list:
```
Comment (jasiu86) [inline tests/functional/object_model/docker.py:109]: Use 'in' statement:
line in lines

**Review by rasapala — APPROVED** [review]
LGTM

**Review by atobiszei — APPROVED** [review]
Agree to have other tests updated afterwards.

**Pull request merged**